example/cumulus-tf/variables.tf

```diff
@@ -350,7 +350,7 @@ variable "rds_admin_access_secret_arn" {
 variable "async_operation_image_version" {
   description = "docker image version to use for Cumulus async operations tasks"
   type = string
-  default = "52"
+  default = "53"
```

Comment:
Note: we need to determine a way to handle this given main is on '52' and if updates are made there to this image, it would go to '54' internally.